### PR TITLE
[khmer_angkor] Fix keyboard_info fonts

### LINKED ISSUE #1996: bug: khmer_angkor.keyboard_info has font instead of oskFont
Compare the results of https://api.keyman.com/cloud/4.0/keyboards?jsonp=keyman.register&languageidtype=bcp47&version=16.0&keyboardid=sil_jarai&timerid=11 with the corresponding keyboard's .kps definitions:

From the KMW Cloud 4.0 API:
```javascript
/// ...
"font":{"family":"Khmer Busra Kbd","source":["khmer_busra_kbd.ttf"]} // no "oskFont" entry
/// ...
```

https://github.com/keymanapp/keyboards/blob/0fcd24373b35acd6ee8386b351512403407a305e/release/k/khmer_angkor/source/khmer_angkor.kps#L137-L146

```xml
<Keyboard>
  <Name>Khmer Angkor</Name>
  <ID>khmer_angkor</ID>
  <Version>1.3</Version>
  <OSKFont>..\..\..\shared\fonts\khmer\busrakbd\khmer_busra_kbd.ttf</OSKFont>
  <DisplayFont>..\..\..\shared\fonts\khmer\mondulkiri\Mondulkiri-R.ttf</DisplayFont>
  <Languages>
    <Language ID="km">Central Khmer (Khmer, Cambodia)</Language>
  </Languages>
</Keyboard>
```

This is affecting how KMW renders text for corresponding keyboards:

image

(Typed via US keys 'sjruk' in order.)

For comparison, everything renders fine on my iPhone while using `khmer_angkor`.  It's KMW that's affected.

### PR DESCRIPTION
**Pull request by LornaSIL**
Fixes #1996

### CODE REVIEW

File: release/k/khmer_angkor/khmer_angkor.keyboard_info [hunk 1]
```diff
@@ -2,10 +2,16 @@
...
 		"km": {
-			"font": {
+			"oskFont": {
 				"family": "Khmer Busra Kbd",
 				"source": [
-					"khmer_busra_osk.ttf"
+					"khmer_busra_kbd.ttf"
+				]
+			},
+		"font": {
```
Comment (mcdurdin) [inline release/k/khmer_angkor/khmer_angkor.keyboard_info:11]:
```suggestion
	  	"font": {
```

**Review by mcdurdin — APPROVED** [review]
LGTM

**Comment by LornaSIL** [comment]
@mcdurdin I guess it needs another review since I accepted your suggested modification.

**Review by mcdurdin — APPROVED** [review]
LGTM

**Pull request merged**